test/setup/main.tf

```diff
@@ -18,15 +18,15 @@ module "project" {
   source  = "terraform-google-modules/project-factory/google"
   version = "~> 3.0"
```

Comment:
nit: we can take this opportunity to bump to latest version of project factory and update the setup providers as well.